app/main.py

```diff
@@ -1,2 +1,90 @@
+from collections.abc import Iterator
+from dataclasses import dataclass
+
+
+@dataclass
+class Node:
+    key: object
+    hash_: object
+    value: object
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.size = 0
+        self.load_factor = 2 / 3
+        self.capacity = 8
+        self.hash_table: list = [None] * self.capacity
+
+    def get_cell(self, key: object) -> int:
+        return hash(key) % self.capacity
+
+    def update(self) -> None:
```

Comment:
Better to name it like `resize` or smth. Because we are chaning size of our dictionary